ex02.exs

```diff
@@ -1,6 +1,37 @@
 
 defmodule Ex02 do
-
+  def new_counter(initial_value \\ 0) do
+    { :ok, pid } = Agent.start_link(fn -> initial_value - 1 end)
+    pid
+  end
+
+  def next_value(pid) do
+    Agent.get_and_update(pid, fn state ->
+      { :ok, state + 1 }
+    end)
+
+    Agent.get(pid, &(&1))
+  end
+
```

Comment:
Unfortunately, the rubric says:

>  Last (for this exercise), we'll create a global counter by adding
  two new functions to Ex02. **These will use an agent to store the
  count**, but how can you arrange things so that you don't need to pass
  that agent into calls to `global_next_value`?

So the idea was that you were to adapt the first exercises in this module to use a named agent.

### -5